packages/salesforcedx-utils-vscode/src/channels/channelService.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import stripAnsi from 'strip-ansi';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+
+import { CommandExecution } from '../cli';
+
+export const DEFAULT_SFDX_CHANNEL = vscode.window.createOutputChannel(
```

Comment:
Per discussions with @lcampos offline, we need to include this in an init function to create the output channel in extension init, since extensions using the utils channel service will have its own channel.